sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/signaling/TrouterUtils.java

```diff
@@ -161,7 +176,10 @@ private static BaseEvent getParticipantsAdded(JSONObject payload) {
                 ChatParticipant chatParticipant = new ChatParticipant();
                 chatParticipant.setUser(communicationUser);
                 chatParticipant.setDisplayName(member.getString("displayName"));
-                chatParticipant.setShareHistoryTime(new Date(member.getString("shareHistoryTime")).toString());
+                chatParticipant.setShareHistoryTime(
+                    OffsetDateTime.ofInstant(
```

Comment:
small suggestion, have a function with this code and reuse it with line 140.
i.e getEpocTimeInUtc(localTime)